zulipterminal/ui_tools/messages.py

```diff
@@ -798,14 +808,59 @@ def update_message_author_status(self) -> bool:
 
         return author_is_present
 
+    @staticmethod
+    def transform_content_alert_words(content: str, alerted_list: List[str]) -> Any:
+        alert_regex_replacements = {
+            "&": "&amp;",
+            "<": "&lt;",
+            ">": "&gt;",
+            # Accept quotes with or without HTML escaping
+            '"': r"(?:\"|&quot;)",
+            "'": r"(?:\'|&#39;)",
+        }
+
+        before_punctuation = r"\s|^|>|[\\(\".,';\\[]"
+
+        after_punctuation = r"(?=\s|$|<|[\\)\"?!:.,';\]!])"
+
+        clean: str
+
+        def replace_callback(match: Match[Union[str, str, str]]) -> str:
+            before = match.group(1)
+            word = match.group(2)
+            after = match.group(3)
```

Comment:
Minor: You could use `match.group(1,2,3)` here, which returns a tuple consisting of the three values.

Reply:
The only use these 3 have is in here:
```
 if in_tag:
                return f"{before}{word}{after}"
            return f"{before}<span class='alert-word'>{word}</span>{after}"
```
 I could have directly used `match. group(n)` but didn't. I think separating them into 3 variables gives better readability
 By following this suggestion of `match.group(1,2,3)`, I need it to be stored in some var and break it using the index to use it in the above code which will not clarify any further of what is being done. 
So why group it when it can be accessed individually without creating any `tuple`

Reply:
If I follow the above correctly, would it help if you used tuple unpacking?
```python3
one, two, three = function_returning_tuple_of three_values()

Reply:
Yep! Multiple assignments in one line can do the same thing.
```
before,word,after = match.group(1,2,3)
```